src/tabs/tabset.component.ts

```diff
@@ -40,6 +40,15 @@ export class TabsetComponent implements OnDestroy {
     this.setClassMap();
   }
 
+  /** provides possibility to set keyNavigations enable or disable, by default is enable */
+  @Input()
```

Comment:
let's not bloat component API, and just use config

Reply:
i changed to use the config, please review it